packages/block-library/src/query-pagination-next/view.js

```diff
@@ -9,21 +9,30 @@ const load = () => {
 	link.addEventListener( 'click', async ( e ) => {
 		e.preventDefault();
 
-		// Fetch the HTML of the new page.
-		const url = e.target.href;
+		const { block: blockName, attribute, newValue } = e.target.dataset;
+		const blockSelector = `.wp-block-${ blockName.replace( 'core/', '' ) }`; // TODO: Need a better mechanism here.
+
+		// Fetch the HTML of the new block.
+		// TODO: Need to include blocks' current attributes.
```

Comment:
Those attributes are well-known and serialized (as part of the Gutenberg block delimiting comment `<!-- wp:something ... -->`). We need to expose them here on the frontend.